src/logger.cpp

```diff
@@ -27,4 +30,5 @@ void Logger::log(LogLevel level, const std::string &message)
     }
 
     logFile << Logger::getCurrentTime() << " " << levelStr << message << std::endl;
+    std::cout << Logger::getCurrentTime() << " " << levelStr << message << std::endl;
```

Comment:
Unnecessary

Reply:
forgot to remove it, will do it, had to use it as when I had the file open it was destroying my pc q.q